README.md

```diff
@@ -1,2 +1,10 @@
 # dev-tools-configs
-This repository contains set of configuration files for development tools used by Dasharo Team members
+This repository contains a set of configuration files for text editors and IDEs
+used by Dasharo Team and their community.  
+
+## Available configurations  - [VS Code](/vscode) 
+
+## Contributions  
+Feel free to contribute the configuration of any editor that you use. You can
+use the template from `.editor_template` folder. Improvements to existing
```

Comment:
@kotylamichal lets name this dir without dot so it will be visible in terminal without `ls -a`. Please, name it `00_template`

Reply:
@kotylamichal please change here folder name to `00_template`

Reply:
done https://github.com/Dasharo/dev-tools-configs/pull/1/commits/bfad4c29c86995972c56d9f4a5e6283342d6f77e